maintainers/maintainer-list.nix

```diff
@@ -8567,6 +8567,12 @@
     githubId = 7669898;
     name = "Katharina Fey";
   };
+  spease = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add spease``.